README.md

```diff
@@ -31,9 +29,7 @@ import {
 } from '@primer/components'
 ```
 
-### Styling
-
-This project uses [emotion] to generate static CSS for most component styles, but still relies on [Primer CSS] for some classname-based styles that haven't yet been ported over. Components that haven't yet been ported over rely on [a subset of Primer CSS](https://github.com/primer/components/blob/master/src/primer-components.scss) that's globally injected at import time.
```

Comment:
I decided to remove this section since the end user doesn't really _need_ to know how our CSS works (since they no longer need to do anything on their end), all they need to know is that styles are included out of the box.